src/commands/pickFuncProcess.ts

```diff
@@ -21,10 +22,17 @@ export async function pickFuncProcess(): Promise<string | undefined> {
     // Start (or restart) functions host (which will also trigger a build)
     await vscode.commands.executeCommand('workbench.action.tasks.runTask', funcHostTaskId);
 
-    const timeoutSeconds: number = 60;
-    const maxTime: number = Date.now() + timeoutSeconds * 1000;
+    const settingKey: string = 'pickFuncProcessTimeout';
```

Comment:
how about functionStartTimeout?  Or at least "find" instead of "pick".

Reply:
So this is what a user's launch.json looks like for C#:
```
{
  "version": "0.2.0",
  "configurations": [
    {
      "name": "Attach to C# Functions",
      "type": "coreclr",
      "request": "attach",
      "processId": "${command:azureFunctions.pickProcess}"
    }
  ]
}
```
You'll notice that it explicitly references the "azureFunctions.pickProcess" command. I would prefer to keep the setting consistent with the command name (which now that I look closer, I realize I should remove 'Func' and change it to "pickProcessTimeout"). In other words, I don't disagree with your comment, but it's too late to change the command name (without breaking existing projects) and I think being consistent is more important. Sound fine to you?

Reply:
Sure